packages/repository/src/__tests__/unit/repositories/legacy-juggler-bridge.unit.ts

```diff
@@ -411,9 +414,26 @@ describe('DefaultCrudRepository', () => {
   });
 
   it(`throws error when execute() not implemented by ds connector`, async () => {
-    const repo = new DefaultCrudRepository(Note, ds);
+    const repo = new DefaultTransactionalRepository(Note, ds);
     await expect(repo.execute('query', [])).to.be.rejectedWith(
       'execute() must be implemented by the connector',
     );
   });
+
+  it('throws an error when beginTransaction() not implemented', async () => {
```

Comment:
These new tests are inside `describe('DefaultCrudRepository')`, but they are testing `DefaultTransactionalRepository` class instead.

Please move them to a new describe block.

```ts
describe('DefaultCrudRepository', () => {
  // existing test suite
});

describe('DefaultTransactionalRepository' => {
 // your new tests calling `repo.beginTransaction'
});
```